### ISSUE TITLE
1.12.2 Mob and player can't be burned or set on fire at all

### ISSUE THREAD

**Issue by zongitgud** [RobertSkalko/Mine-and-Slash#87]
Version: 1.12.2-1.7.7
Weapon, skill, magic or enchantment can't set any living thing on fire. Lava can still burn you (show particle) but as soon as you move out of the lava you won't get burn any more  (normally you'll still getting burn for a couple of second). Blaze, Ghast and walking on fire can't set you on fire at all. This made weapon and enchantment like Ice and Fire: Dragon sword or Fire aspect, Fire Protection become useless, and pretty much made any mob that fire related less dangerous.
This was not cause by other mod as i have specific disable them to see which one causing it and it was M&S :c Even start a new game with only M&S still can't burn anyone

**Comment by RobertSkalko** [comment]
@AzureZhen i think i made this in onevent somewhere, probably to make them not burn in daylight but didn't really figure the problems. Ah right! The reason for it was that mobs in adventure maps can spawn in both day and night, and they'd burn in day which was undesirable. To fix it, make it only target mobs and only if they are in adventure maps. The other solution would be what i did in 1.14, make adventure maps always night time.

**Comment by zongitgud** [comment]
Actually Nightime adventure maps sound pretty neat. Spooky...

**Comment by AzureDoom** [comment]
Fixed and closed. Made this a configurable option. Just the burn damage call.

**Comment by RobertSkalko** [comment]
```
if (ModConfig.Server.SHOULD_MOBS_BURN) {
					event.getEntityLiving().extinguish();
```

if burns, dont?

**Comment by AzureDoom** [comment]
Ah shit, reverse logic. Good check, will fix when in a few hours.

**Comment by AzureDoom** [comment]
Fixed in 1.9.2
